docs/infrastructure/microk8s-quickstart.md

```diff
@@ -80,31 +77,22 @@ Now that MicroK8s is running on the machine we can proceed to enabling the neces
     microk8s.status --wait-ready
     ```
 
-4.  Install version v2.16.3 of the Helm client:
-
-    ```bash
-    sudo snap install helm --classic --channel=2.16/stable
-    ```
-
-5.  Install the Helm server:
+4.  Install Helm vervsion 3.2.0:
 
     ```bash
-    microk8s.kubectl create serviceaccount --namespace kube-system tiller
-    microk8s.kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
-    helm init --service-account tiller
+    sudo snap install helm --classic --channel=3.2/stable
     ```
 
-6.  The addons are now enabled and the MicroK8s instance bootstrapped. However, we must wait for some MicroK8s pods to be ready, as failing to do so can result in the pods entering a `CrashLoopBackoff` state:
+5.  The addons are now enabled and the MicroK8s instance bootstrapped. However, we must wait for some MicroK8s pods to be ready, as failing to do so can result in the pods entering a `CrashLoopBackoff` state:
 
     ```bash
     microk8s.kubectl wait -n kube-system --for=condition=Ready pod -l k8s-app=kube-dns
     microk8s.kubectl wait -n kube-system --for=condition=Ready pod -l k8s-app=hostpath-provisioner
     # If the following command fails, you probably installed the wrong MicroK8s version
     microk8s.kubectl wait -n default --for=condition=Ready pod -l name=nginx-ingress-microk8s
-    microk8s.kubectl -n kube-system wait --for=condition=Ready pod -l name=tiller
```

Comment:
I believe this line is no longer necessary. Please check.

Reply:
Its no longer required.